<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoistureDialog</class>
 <widget class="QDialog" name="MoistureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Moisture Data</string>
  </property>
  <widget class="QLabel" name="lineData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Reading Time:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePercentStr">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Percentage Moisture</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnter">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFinish">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Finsh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBackward">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnForward">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>70</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;F</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lePercentStr</tabstop>
  <tabstop>btnEnter</tabstop>
  <tabstop>btnFinish</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>btnBackward</tabstop>
  <tabstop>btnForward</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
